MODULE.bazel

```diff
@@ -886,3 +891,98 @@ mainnet_versions(
     name = "mainnet_versions",
     path = "//:mainnet-subnet-revisions.json",
 )
+
+# Set up hermetic cc toolchains for binaries and canisters
+toolchains = use_extension("@hermetic_cc_toolchain//toolchain:ext.bzl", "toolchains")
+toolchains.extra_settings(settings = ["//bazel:use_hermetic_cc"])
```

Comment:
can you add a note about what `extra_settings` does, or better yet, create a ticket upstream and link it here and/or from the patch?

Reply:
Comments with the upstream PR added here and to the patch file 👍.